images/chromestage/go.mod

```diff
@@ -19,3 +20,5 @@ require (
 	golang.org/x/net v0.17.0 // indirect
 	golang.org/x/sys v0.13.0 // indirect
 )
+
+replace github.com/ajbouh/substrate/pkg/commands => ../../pkg/commands
```

Comment:
@ajbouh I copied the pattern used for including `cueloader` here. This isn't needed when working locally with the `go.work` file, but was necessary for the Dockerfile build.